prisma/schema.prisma

```diff
@@ -0,0 +1,22 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+
+}
+
+datasource db {
+  provider = "sqlite"
+  url      = "file:./dev.db"
+}
+
+model Agencies{
```

Comment:
Typically the name of a model is Singular: so Agency
Think of what you are describing in your Model as being 1 row